src/components/Home/Home.js

```diff
@@ -55,20 +49,14 @@ class Home extends React.Component {
 					<div className={styles['body']}>
```

Comment:
We can move the vehicles check before iterating the vehicles and then display the **_Add Link_** inside. So the duplicated code  can be removed

> 
<div className={styles['cta']}>
   <Link to={getRouteDetails(routeConstants.ADD_VEHICLE_DETAILS).path}>Add</Link>
</div>

>

Reply:
done